schemas/stsci.edu/asdf/0.1.0/asdf-schema.yaml

```diff
@@ -0,0 +1,46 @@
+%YAML 1.1
+---
+$schema: "http://json-schema.org/draft-04/schema"
+id: "http://stsci.edu/schemas/asdf/0.1.0/asdf-schema"
+title:
+  ASDF schema
+description: |
+  A metaschema extending YAML Schema and JSON Schema to add support
+  for some ASDF-specific checks, related to nd-arrays.
```

Comment:
I think if third parties want to add validators for their custom types, they are still free to do so -- they will simply be ignored by our validator, but it doesn't make those custom types unusable by tools that don't understand them.

The pyasdf implementation of this actually makes it easy for third-party code to add validators to the system (as part of the type definition).  That only applies to pyasdf, of course.

In any event, I agree with you -- since this is so strongly tied to a core type, it's probably fine to include.